### DISCUSSION TITLE
#39 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Riely007** [tog-ETH-er/togETHer#15] [General]
Agenda:

1. Community updates
2. Fireside discussion

> 会议主题：togETHer
> 会议时间： 21:00-22:00
> 重复周期：每两周 (周一)
> 
> 点击链接入会，或添加至会议列表：
> https://meeting.tencent.com/s/LxNajKWJw4RV
> 
> 会议 ID：891 5098 6775

**Comment by Riely007** [comment]
### 以太坊翻译小组

1. 官网翻译和 peer review 进展中
2. 准备翻译 seminar，翻译小组 <> ETHPlanet

思考：
1. 中文 glossary 
2. ethereum.org 教育页面的内容

**Comment by 0x-stan** [comment]
# Dapp-Learning

## 外部协同

### Arbitrum Sharing meetup

5月16日 晚8点 邀请Arbitrum官方来做技术分享，欢迎参加～

腾讯会议号：665-488-560


## repo update

12个PR #737 ～ #748

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## basic task translate

开始针对前29个基础任务的README进行中转英翻译，已经召集6位小伙伴分工合作，进度50%

## sharing meeting

近两周举办了3场分享会

- 《zkEVM》by Ye Zhang  (ScrollTech)
  - https://www.youtube.com/watch?v=SEp5SFaYQHY
  - https://www.youtube.com/watch?v=Hc9gb3nlK9k

- 《PoS和Tendermint共识讲解》 by Qi Zhou
  - https://www.youtube.com/watch?v=FiOTpYXyP2o
  - https://www.youtube.com/watch?v=J34dCMIMxd4

- 《Terra and Anchor》by Ann
  - https://www.youtube.com/watch?v=GX8IqpJfT-Y  

- 《Defi模型的最优化理论》by Guangwu Xie
  - https://www.youtube.com/watch?v=4n_059bz1Kc

## 翻译小组

- 正式发布14集
  - https://mp.weixin.qq.com/s/LYQjj5PEY8xIf8nbhroqDw

**Comment by bambalados** [comment]
**CryptoTechNight:**

本周四(05.19)将举办CTN#13, 作为一个ETH Shanghai的side event。

> 这一期的主题会环绕着Layer 2， 邀请的嘉宾来自Arbitrum 和 Polygon

**Comment by stephanieeee-ho** [comment]
### ECN
1. 常规的翻译和周报输出，组织了一次社区翻译活动
2. 举办了第一场合并主题活动，现在在准备下一场跟 rocket pool 的活动，欢迎合作

**Comment by liushooter** [comment]
### Rebase

- [Web3日报](https://rebase.network/posts/category/web3daily) 正常更新 14 期
- 更新[多篇招聘](https://github.com/rebase-network/who-is-hiring)
- 更新多篇[活动介绍](https://github.com/rebase-network/event-pool)

**Comment by winkybupt** [comment]
# 原语里弄
1. 准备 EthShanghai 主持工作
2. 各课题稳步推进，预计6月发布 1-2 篇。

**Comment by Emilyxiannv** [comment]
### DAOSquare
1、做了一次关于创作者经济这个话题的分享和讨论：https://www.youtube.com/watch?v=Q349HabolZc

**Comment by bobjiang** [comment]
- GitcoinDAO s14 budget proposal https://gov.gitcoin.co/t/bobs-voter-guide-season-14/10575
- GTC utility https://gov.gitcoin.co/t/gtc-conviction-voting-dapp-gtc-utility/10523
- translation coordinations for GreenPilled
- EthShanghai speech

**Comment by Riely007** [comment]
### TODO

1. connect ETHTaipei 社区 <>  togETHer
